src/TLMessage/TLMessage/ClientMessages/register_device_push.php

```diff
@@ -8,38 +8,56 @@
 
 namespace TelegramOSINT\TLMessage\TLMessage\ClientMessages;
 
-use Exception;
 use TelegramOSINT\Exception\TGException;
 use TelegramOSINT\TLMessage\TLMessage\Packer;
 use TelegramOSINT\TLMessage\TLMessage\TLClientMessage;
 
 /**
  * @see https://core.telegram.org/method/account.registerDevice
- * current has ctor 0x68976c6f (layer 105)
  */
 class register_device_push implements TLClientMessage
 {
-    private const CONSTRUCTOR = 1669245048; // 0x637EA878
+    private const CONSTRUCTOR = 3968205178; // 0xec86017a
 
     public function getName(): string
     {
         return 'register_device_push';
     }
 
     /**
+     * @param int $length
+     *
      * @throws TGException
      *
      * @return string
      */
-    public function toBinary(): string
+    private static function createSecret(): string
     {
-        try {
-            return
-                Packer::packConstructor(self::CONSTRUCTOR).
-                Packer::packInt(7).
-                Packer::packString(random_int(1000, 9999).random_int(1000, 9999).random_int(1000, 9999).random_int(1000, 9999).random_int(100, 999));
-        } catch (Exception $e) {
+        /** @noinspection CryptographicallySecureRandomnessInspection */
+        $secret = openssl_random_pseudo_bytes(256, $strong);
+        if ($secret === false || $strong === false) {
             throw new TGException(TGException::ERR_CRYPTO_INVALID);
         }
+
+        return $secret;
+    }
+
+    /**
+     * @throws TGException
+     *
+     * @return string
+     */
+    public function toBinary(): string
+    {
+        return
+                Packer::packConstructor(self::CONSTRUCTOR).
+                Packer::packInt(0). // flags , no_muted=false
+                Packer::packInt(2). // token_type, 2 - FCM (firebase token for google firebase)
+                Packer::packString('eJw6Qq8aGB8:APA91bGmp2tE0mGtW2zs-zk-rOoOAV_ggVeDCAojNuUd3A4Elf47vJKUSWClmJEAb6xae2wuIGZzVSwLvptEmQ9lPZIevuTf_Jbo3RMdXXq_Tebcfh3r9ioVhYRqkGH-sygjXj9e6YES'). // token
```

Comment:
Bad idea to hardcode it!

Reply:
It was already in register_device_push_java.php

Reply:
Kick off this class